internal/build/download.go

```diff
@@ -82,6 +82,7 @@ func (db *ChecksumDB) DownloadFile(url, dstPath string) error {
 	fmt.Printf("%s is stale\n", dstPath)
 	fmt.Printf("downloading from %s\n", url)
 
+	// #nosec (internal tool nothing to worry about)
```

Comment:
What is the warning? As a reason, "internal tool" is tricky because if it exposes the dev machines, that could be very bad.

Reply:
The warning was about doing `http.Get(url)` where `url` is not sanitized and comes from the user.

The rule is: **G107: Url provided to HTTP request as taint input** ([rules here](https://github.com/securego/gosec#available-rules))

Reply:
Ok, that makes sense 👌 . A comment that is more specific would be helpful.
```suggestion
	// #nosec (Ignoring sanitization warning. Intended to be an arbitrary url.)
```